### PR TITLE
fix: removing ticket group, some extra line and adding a popup

### LINKED ISSUE #4623: Tickets: Take out "Ticket Group" from Ticket Settings and add pop-up to "Hide Ticket" instead
The ticket group option makes sense logically, but users who are new to the system do not understand it even though it is similar to Eventbrite. The ticket group option in the ticket settings allows organizers to define ticket access through a specific code.

### Please change the feature as follows:
* Take out the ticket group option UI from the ticket settings
* Add a pop-up to when the user clicks "Hide Ticket"
* The pop-up text states the following: "To sell hidden tickets you need to set up an access code under Event Dashboard > Tickets Tab > Access Code. [OK] [Cancel]"
  * If the user chooses "OK" add the tick in the tick box
  * If the user chooses "Cancel" do not add a tick in the box
* Change text "Ticket would still be available for sale to Organizers." -> "To sell hidden tickets you need to set up an access code under Event Dashboard > Tickets Tab > Access Code.".

### Test
**Please test the feature**
1. Create an event with a hidden ticket.
2. Check that the hidden ticket does not show up on the public event pages.
3. Create an access code for the hidden ticket type.
4. Apply the access code to the public event page.
5. Order a hidden ticket on the event page, where you use the access code.

**Report any issues you encounter in the process.**

Screenshot from 2020-07-23 19-45-41

### LINKED ISSUE #5095: Wizard Step 1: Ticket box - take out "Settings"
The word "Settings" is not needed in the ticket box. Please take out it and move the below content up.

Screenshot from 2020-09-19 14-50-11

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #4623
Fixes #5095
@iamareebjamal , when i am clicking no in confirm modal, it is executing action, and on yes it's not. means that in this, yes is working as no, and no is working as yes. rest is working correctly.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
captured (2)

**Comment by mariobehling** [comment]
Thanks. For the pop up font, style and size please use the same styles as it is used for other pop ups elsewhere.

**Comment by mariobehling** [comment]
It seems the pop up functionality is inverted. Instead of yes/no please use OK/Cancel.

* [x] The pop-up text states the following: "To sell hidden tickets you need to set up an access code under Event Dashboard > Tickets Tab > Access Code. [OK] [Cancel]"
  * [ ] If the user chooses "OK" add the tick in the tick box
  * [ ] If the user chooses "Cancel" do not add a tick in the box

**Comment by maze-runnar** [comment]
> Instead of yes/no please use OK/Cancel.

@iamareebjamal how to change button text in confirm helper?

**Comment by iamareebjamal** [comment]
Make them props

**Comment by maze-runnar** [comment]
how to use them in helper

**Comment by iamareebjamal** [comment]
Currently they'll be hardcoded. You have to make them props and pass them in the component

**Comment by iamareebjamal** [comment]
See other dialogs which are using OK/Cancel

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/helpers/confirm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6200188&bid=20223988)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/helpers/confirm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6200188&bid=20223988)

**Comment by maze-runnar** [comment]
Screenshot from 2020-09-18 16-40-35

**Comment by mariobehling** [comment]
> For the pop up font, style and size please use the same styles as it is used for other pop ups elsewhere.

**Comment by mariobehling** [comment]
* [ ] Good that you took out the line breaks, but we also need to keep some spacing in the area where the breaks used to be. So, yes get rid of the breaks, but keep a bit of spacing in the area, please.
* [ ] One of the PRs recently introduced a line break below "Yes, accept payment through Stripe. Please also get rid of that line break and make it look the same as the above area of Paypal. 
Screenshot from 2020-09-18 20-41-08

**Comment by maze-runnar** [comment]
Screenshot from 2020-09-20 17-12-09

File: app/helpers/confirm.js [hunk 1]
```diff
@@ -9,7 +9,12 @@ export default Helper.extend({
 
   compute(params) {
     return () => {
-      if (params.length >= 2) {
+      if (params.length >= 6) {
+        this.confirm.prompt("Are You Sure?", { 'denyText': params[2], 'approveText': params[3], 'denyColor': params[4], 'approveColor': params[5], 'extra': params[0] })
```
Comment (niranjan94) [inline app/helpers/confirm.js:13]: Codacy found an issue: [Strings must use singlequote.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6200188)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- app/helpers/confirm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6200188&bid=20223988)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/helpers/confirm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6200188&bid=20223988)

**Comment by maze-runnar** [comment]
Screenshot from 2020-09-20 18-58-24

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/helpers/confirm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6200188&bid=20223988)

**Review by mariobehling — APPROVED** [review]
Looks good for me UI wise. @iamareebjamal Please check code. Thank you.

File: app/helpers/confirm.js [hunk 2]
```diff
@@ -9,7 +9,12 @@ export default Helper.extend({
 
   compute(params) {
     return () => {
-      if (params.length >= 2) {
+      if (params.length >= 6) {
+        this.confirm.prompt('Are You Sure?', { 'denyText': params[2], 'approveText': params[3], 'denyColor': params[4], 'approveColor': params[5], 'extra': params[0] })
+          .then(() => {
+            params[1]();
+          });
+      } else if (params.length >= 2) {
```
Comment (iamareebjamal) [inline app/helpers/confirm.js:17]: Can you explain please what is happening here. Looks very confusing

Comment (maze-runnar) [reply app/helpers/confirm.js:17]: `this.confirm.prompt('Are You Sure?', { 'denyText': params[2], 'approveText': params[3], 'denyColor': params[4], 'approveColor': params[5], 'extra': params[0] })`

the options are from service/confirm.js. In confirm-moda.js these are props (denyText, approveText...). in confirm modal helper i am using those and in component passing them.

**Pull request merged**